OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -388,7 +382,7 @@ getPointForceDirections(const SimTK::State& s,
 /* add in the equivalent spatial forces on bodies for an applied tension 
     along the GeometryPath to a set of bodyForces */
 void GeometryPath::addInEquivalentForces(const SimTK::State& s,
-    const double& tension, 
+    double tension,
```

Comment:
Unrelated API change?

Reply:
Sort of. When I wrote the virtual method `addInEquivalentForces` I made this `double` out of habit, and then updated this to follow. I vaguely remember reading somewhere that `const` references versus copies for simple types weren't that important, but I could be wrong. I'm fine with reverting just for the sake of not changing the function signature.

Reply:
For trivial types (esp those that fit in a register) it's almost always best to pass by value, yeah, but I'd put any API change in a separate PR if possible

Reply:
Sounds good! I'll revert the change and we can revisit later if needed.